src/Widgets/PagesList.vala

```diff
@@ -291,6 +294,7 @@ public class ENotes.PagesList : Gtk.Box {
                 var page_item = added_pages.get ((int) page.id);
                 page_item.page = page;
                 page_item.load_data ();
+                listbox.invalidate_sort ();
```

Comment:
Since we're removing the page sorting change from this PR, this shouldn't be needed for now :)